code/modules/mob/living/carbon/xenomorph/Castes/Defiler.dm

```diff
@@ -0,0 +1,333 @@
+/datum/xeno_caste/defiler
+	caste_name = "Defiler"
+	display_name = "Defiler"
+	upgrade_name = "Young"
+	caste_desc = "A frightening looking, bulky xeno that drips with suspect green fluids."
+	caste_type_path = /mob/living/carbon/Xenomorph/Defiler
+	tier = 3
+	upgrade = 0
+
+	// *** Melee Attacks *** //
+	melee_damage_lower = 30
+	melee_damage_upper = 35
+
+	// *** Tackle *** //
+	tackle_damage = 40
+
+	// *** Speed *** //
+	speed = -1
+
+	// *** Plasma *** //
+	plasma_max = 400
+	plasma_gain = 20
+
+	// *** Health *** //
+	max_health = 225
+
+	// *** Evolution *** //
+	upgrade_threshold = 400
+
+	deevolves_to = /mob/living/carbon/Xenomorph/Carrier
+
+	// *** Flags *** //
+	caste_flags = CASTE_CAN_BE_QUEEN_HEALED|CASTE_EVOLUTION_ALLOWED|CASTE_CAN_BE_GIVEN_PLASMA
+
+	// *** Defense *** //
+	armor_deflection = 30
+
+	// *** Defiler Abilities *** //
+
+/datum/xeno_caste/defiler/mature
+	upgrade_name = "Mature"
+	caste_desc = "A frightening looking, bulky xeno that drips with suspect green fluids. It looks a little more dangerous."
+
+	upgrade = 1
+
+	// *** Melee Attacks *** //
+	melee_damage_lower = 35
+	melee_damage_upper = 45
+
+	// *** Tackle *** //
+	tackle_damage = 45
+
+	// *** Speed *** //
+	speed = -1.1
+
+	// *** Plasma *** //
+	plasma_max = 500
+	plasma_gain = 25
+
+	// *** Health *** //
+	max_health = 275
+
+	// *** Evolution *** //
+	upgrade_threshold = 800
+
+	// *** Defense *** //
+	armor_deflection = 40
+
+	// *** Defiler Abilities *** //
+
+/datum/xeno_caste/defiler/elder
+	upgrade_name = "Elder"
+	caste_desc = "A frightening looking, bulky xeno that drips with suspect green fluids. It looks pretty strong."
+
+	upgrade = 2
+
+	// *** Melee Attacks *** //
+	melee_damage_lower = 40
+	melee_damage_upper = 45
+
+	// *** Tackle *** //
+	tackle_damage = 50
+
+	// *** Speed *** //
+	speed = -1.7
+
+	// *** Plasma *** //
+	plasma_max = 550
+	plasma_gain = 28
+
+	// *** Health *** //
+	max_health = 300
+
+	// *** Evolution *** //
+	upgrade_threshold = 1600
+
+	// *** Defense *** //
+	armor_deflection = 47
+
+	// *** Defiler Abilities *** //
+
+/datum/xeno_caste/defiler/ancient
+	upgrade_name = "Ancient"
+	caste_desc = "Being within mere eyeshot of this hulking, dripping monstrosity fills you with a deep, unshakeable sense of unease."
+	ancient_message = "You are the ultimate alien rapist. You will impregnate the marines, see them burst open before you, and hear the gleeful screes of your larvae."

```

Comment:
i am uncomfortable

Reply:
I did miss this part but, dear revioli god, tone the acient message flavour down and use euphisms, just a little, or I'm giving marines pepper sprays.

Reply:
Literally no one sees it but the player; calm your tits

Reply:
Understood, after all you aren't wrong.

Reply:
Ghommie, chill out, please jesus christ

Reply:
Already done, apologises.

Reply:
even if only the defiler player sees it, someone's going to end up with a rule 8 at some point because of this, i guarantee it